### ISSUE TITLE
how to re-produce the result with only 8-GPUs

### ISSUE THREAD

**Issue by youngwanLEE** [hustvl/MIMDet#3]
Hi,

I'm really impressed by your excellent work.

I have some questions about re-producing results on an 8-GPUs environment.

Could you advise how to adjust hyper-parameters(e.g., learning rate) for an 8-GPUs environment?

And 

Have you already obtained results in an 8-GPU environment?

Thanks in advance :)

**Comment by Yuxin-CV** [comment]
Hi @youngwanLEE, thanks for your interest in our work :)

We actually start with 8 GPUs!

I think it is OK to directly run MIMDet w/ the 25% sampling rate even without grad checkpointing using bsz = 16 (lr with linear scale rule), similar to the Swin config (I think you should be very familiar with that :), it takes about 1 day to finish the 3x training. 

If you use the 50% sampling rate, I guess it will take 3~4 days to finish the training w/ grad checkpointing using 8x 32G V100


We will try to provide a 8x GPUs config with strong result soon to make our work more accessible to the community.

**Comment by youngwanLEE** [comment]
@Yuxin-CV 

Thanks for the quick reply!
I hope you're ready for releasing the 8x GPUs config ASAP!!

**Comment by Yuxin-CV** [comment]
Hi @youngwanLEE, we have updated [a new training config](https://github.com/hustvl/MIMDet/blob/main/configs/mimdet/mimdet_vit_base_mask_rcnn_fpn_sr_0p25_800_1333_4xdec_coco_3x_bs16.py) w/ sample ratio = 0.25 for 8x GPUs (bsz = 16) environment.

The results (49.9 Box AP / 44.6 Mask AP) match our default settings (49.9 Box AP / 44.7 Mask AP), and are better than the Swin-Base counterpart (49.2 Box AP / 43.5 Mask AP) under a similar total training time (~2d6h).

**Comment by youngwanLEE** [comment]
@Yuxin-CV 
Thanks !! 
But, I just checked the  46.52 Box AP / 41.61 Mask AP in your [log](https://github.com/hustvl/Storage/releases/download/v1.1.1/mimdet_vit_base_mask_rcnn_fpn_sr_0p25_800_1333_4xdec_coco_3x_bs16.json
).

Could you check again if I'm wrong?

**Comment by Yuxin-CV** [comment]
> @Yuxin-CV Thanks !! But, I just checked the 46.52 Box AP / 41.61 Mask AP in your [log](https://github.com/hustvl/Storage/releases/download/v1.1.1/mimdet_vit_base_mask_rcnn_fpn_sr_0p25_800_1333_4xdec_coco_3x_bs16.json).
> 
> Could you check again if I'm wrong?

Hi, during inference, we use sample ratio = 1.0 instead of 0.25. see Table 2 in our paper for the details.

**Comment by youngwanLEE** [comment]
@Yuxin-CV 
Oh I see.  
In your [inference](https://github.com/hustvl/MIMDet/blob/main/README.md#inference) instruction, I can find it.
`model.backbone.bottom_up.sample_ratio=1.0`

Thanks again :)

**Comment by Yuxin-CV** [comment]
> @Yuxin-CV Oh I see. In your [inference](https://github.com/hustvl/MIMDet/blob/main/README.md#inference) instruction, I can find it. `model.backbone.bottom_up.sample_ratio=1.0`
> 
> Thanks again :)

I hope our work is helpful to you :)

**Comment by normster** [comment]
Hi @Yuxin-CV,

Would you happen to have a plain ViT training recipe handy for 8 GPUs available?

Thanks!
